src/map/packets/action.cpp

```diff
@@ -100,6 +100,8 @@ CActionPacket::CActionPacket(action_t& action)
         ref<uint8>(0x0C) = 0x1C;
         ref<uint8>(0x0D) = 0x5D;
         ref<uint8>(0x0E) = 0x19;
+
+        ActionType = ACTION_WEAPONSKILL_START;
```

Comment:
I take exception to assigning the action type of [weaponskill start](https://github.com/Windower/Lua/wiki/Action-Category-03) inside a [mobskill interrupt](https://github.com/Windower/Lua/wiki/Action-Category-07) case.

But we apparently use `ACTION_WEAPONSKILL_START` for mobskill starts too, so I'm just going to throw my hands in the air and pretend I didn't see this. 🤷